intl/messages/en-US.json

```diff
@@ -1,5 +1,10 @@
 {
     "home.title": "Homepage",
+
```

Comment:
It seems that internal server error might not be always correct as this will be used for client only errors as well. Maybe we can make it more generic?

Reply:
Maybe `An error has occurred`?

Reply:
Yep sounds good!